crates/test_probe/src/main.rs

```diff
@@ -0,0 +1,30 @@
+use composer::api::{Client, Event};
+use eyre::Result;
+use std::{env, thread, time::Duration};
+
+fn main() -> Result<()> {
+    color_eyre::install()?;
+
+    // Get server address, if given
+    let args: Vec<String> = env::args().collect();
+    let client = match args.get(1) {
```

Comment:
Nit: this compiles for me:
```suggestion
    let client = match env::args().skip(1).next() {
```

Reply:
I [plan](https://github.com/tonarino/acoustic_profiler/issues/16) to replace this with `clap` anyway, maybe let's not bother with it now. .)

Reply:
Fair enough, this was just an exercise (to get rid of `collect()`). BTW `clippy` told me to use `.nth(1)` instead of `skip(1).next()`.

Reply:
what clippy command / version do you use? Because mine didn't report anything. Maybe I should update my toolchain.

Reply:
@goodhoko the one from Rust 1.69, but I think 1.68 also issued a warning about this:
```
strohel@thicky ~/work/acoustic_profiler $ cargo clippy
warning: called `skip(..).next()` on an iterator
 --> crates/test_probe/src/main.rs:9:35
  |
9 |     let client = match env::args().skip(1).next() {
  |                                   ^^^^^^^^^^^^^^^ help: use `nth` instead: `.nth(1)`
  |
  = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#iter_skip_next
  = note: `#[warn(clippy::iter_skip_next)]` on by default

warning: `test_probe` (bin "test_probe") generated 1 warning (run `cargo clippy --fix --bin "test_probe"` to apply 1 suggestion)
    Finished dev [unoptimized + debuginfo] target(s) in 0.03s
strohel@thicky ~/work/acoustic_profiler $ cargo clippy --version
clippy 0.1.69
```